src/Extend/Formatter.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * (c) Toby Zerner <toby.zerner@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Flarum\Extend;
+
+use Flarum\Extension\Event\Disabled;
+use Flarum\Extension\Event\Enabled;
+use Flarum\Extension\Extension;
+use Flarum\Formatter\Event\Configuring;
+use Flarum\Formatter\Formatter as ActualFormatter;
+use Illuminate\Contracts\Container\Container;
+use Illuminate\Events\Dispatcher;
+
+class Formatter implements ExtenderInterface
+{
+    protected $callback;
+
+    public function configure(callable $callback)
+    {
+        $this->callback = $callback;
+
+        return $this;
+    }
+
+    public function __invoke(Container $container, Extension $extension = null)
```

Comment:
I've seen this in Franz' code too. He was referring to `$laravel` in some situations. In the end they are all `$app` and it doesn't make sense to differ the naming all over our code. Any reason for resolving `Container` here instead of `Application`?

Reply:
This is the extender interface. We're dealing with a container (contract) instance here, not an "app".

Reply:
On that note, if an extender needed to access the Application instance, would it be expected to retrieve it from the Container? Is there an argument that we should be passing Application into extenders?

Reply:
I am not a big fan of extending `Application` from `Container` anyway, I might change that. So yes, you'd have to resolve it.

Reply:
Separating `Application` from `Container` sounds good to me. :+1: